### DISCUSSION TITLE
Need to understand few things

### DISCUSSION THREAD

**Discussion by nofx77** [orestbida/cookieconsent#583] [General]
Correct me if im incorrect but in the documentation it says : 

**You can configure the plugin so that it automatically clears existing cookies when a category is rejected/disabled.

Cookies must be listed manually as the plugin is not aware of them.**

But if thoses cookies are from another domain let say youtube the plug-in cannot delete them even if you list them because they are Cross-Origin cookies right?

**Comment by orestbida** [comment]
Yes, you can only delete first party cookies, those set on your domain.
